.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
   "cSpell.words": [
+    "formik",
```

Comment:
I'm adding these words to the vscode workspace so that the IDE quits nagging when someone names a package something silly, like "formik". Is this beneficial?

Reply:
@JCarmine generally I'd recommend keeping any editor-specific settings/files out of the repo, and making these changes only locally while they stay in gitignore. I get it's probably annoying but it feels inherently personal to the user.